v4_MIGRATION_GUIDE.md

```diff
@@ -48,6 +49,8 @@ In the `AndroidManifest.xml`, migrate the `intent-filter` from your v3 integrati
 </activity>
 ``` 
 
+Note: `android:exported` is required if your app compile SDK version is API 31 (Android 12) or later.
+
```

Comment:
Should this go in the README instead because it also affects merchants onboarding directly to v4?

Reply:
We don't have anything in the README currently showing how the deep-link intent-filter should be configured in the `AndroidManifest`. We have a JIRA ticket open to update the dev docs with the new flag. Do you think we should also add a section to the README about Android 12?

Reply:
As long as we have a JIRA to make this more explicit, that is OK. Ideally it is called out elsewhere than just the migration guide.